programs/ziti-edge-tunnel/include/model/dtos.h

```diff
@@ -92,7 +92,8 @@ XX(StartTime, timestamp, none,StartTime, __VA_ARGS__) \
 XX(Identities, tunnel_identity, array, Identities, __VA_ARGS__)
 
 #define INSTANCE_STATUS(XX, ...) \
-XX(ok, __VA_ARGS__)
+XX(ok, __VA_ARGS__) \
```

Comment:
Sorry to keep coming back to this. I'm just trying to understand why we have an enum here.

1. What is the difference between the "ok" status and the newly added "initialized" status.
2. Why not use a boolean for the "Status" field of the tunnel_identity model instead of this enum?

Reply:
will remove this enum, I thought, if we need to add any other status in future , we could use this enum

Reply:
We fetch the identities records from the config file and then load the identity files. If the identity file is deleted by any chance, then we dont have to send the information of that identity to the UI when the UI is connected. so I added this status field to identify whether the identity file is found in the config location or not. I am making field boolean now